lib/fog/core/model.rb

```diff
@@ -21,16 +21,7 @@ def initialize(new_attributes = {})
     end
 
     def inspect
-      Thread.current[:formatador] ||= Formatador.new
-      data = "#{Thread.current[:formatador].indentation}<#{self.class.name}"
-      Thread.current[:formatador].indent do
-        unless self.class.attributes.empty?
-          data << "\n#{Thread.current[:formatador].indentation}"
-          data << self.class.attributes.map { |attribute| "#{attribute}=#{send(attribute).inspect}" }.join(",\n#{Thread.current[:formatador].indentation}")
-        end
-      end
-      data << "\n#{Thread.current[:formatador].indentation}>"
-      data
+      Fog::Formatador.new(self)
```

Comment:
I concur and I'll push that in a minute. Are you in favor of having a single formatador instance for the module and just re-initializing at the end of the inspect method or do you think a new formatador instance should be created per call?